backend/internal/controller/adapter/shot_message_test.go

```diff
@@ -63,9 +63,8 @@ func TestUnpackShotMsg(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			var rawSlice []json.RawMessage
 			jsn, err := json.Marshal(tt.msgIn)
-			if err != nil {
-				assert.FailNow(t, err.Error())
-			}
+
+			assert.Nil(t, err)
```

Comment:
```suggestion
			if err != nil {
				assert.FailNow(t, err.Error())
			}
```
I would revert this change. Further tests are dependent on Marshal() method. If this method fails I don't see a point in checking the rest of the tests.

Reply:
Use [`require.Nil`](https://pkg.go.dev/github.com/stretchr/testify@v1.8.0/require#Nil)

Reply:
Done